packages/web/src/lib/web-mode.ts

```diff
@@ -269,4 +269,46 @@ export class WebMode {
 		}
 		throw new Error("Transaction Error");
 	}
+
+	async makeTx(transactions: ExecParams[]): Promise<Transaction[]> {
```

Comment:
this function shouldn't be async. Let's add required parameter: `sp: algosdk.SuggestedParams`
Also let's rename the first argument, I propose the following interface:
```
makeTx(execParams: ExecParams[], txParams: algosdk.SuggestedParams): Transaction[]
```

Reply:
Do we want to change it in the rest of the functions as well. For example `makeAndSendTx` should we also require the `suggestedParams` ?

Reply:
Yes, otherwise this functions are less usable